### PR TITLE
Jsmr search display

### PR DESCRIPTION
**Pull request by ecejmsr**
First pull request. I implemented the search view. The view has a recycler view that also has a fragment layout that I made. It also contains a search view that has a bit of padding underneath to show some segregation between the search bar and the first item of the list.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">FInalProject</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (itzcoatl90) [inline app/src/main/res/values/strings.xml]: Hey.... can we remove this string? this was attached as part of default code... everything else is ok :)

**Review by itzcoatl90 — CHANGES_REQUESTED** [review]
1 comment....

**Comment by ecejmsr** [comment]
I deleted parts needed from .xml

**Review by itzcoatl90 — CHANGES_REQUESTED** [review]
Please solve conflicts on branch comparing the lasts merges

**Pull request closed without merging**